### PR TITLE
completed part 4.

### PR DESCRIPTION
**Pull request by Lockless404**
Added owners and species tables. Inserted owners. Added species. manipulated data using JOIN.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -13,3 +13,24 @@ weight_kg decimal,
...
+PRIMARY KEY (id)
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INT; 
+ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species (id);
+
+ALTER TABLE animals ADD COLUMN owner_id INT;
+ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
```
Comment (DammyShittu) [inline schema.sql:36]: [OPTIONAL]

- Awesome approach to altering the animals table. 👏 
What do you think about adding these columns and also making them foreign keys in two lines of code(one for each column)? Shorter, right? 😀 

You can do it like this 👇 

```
ALTER TABLE animals ADD species_id int REFERENCES species(id);

ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

**Review by DammyShittu — APPROVED** [review]
Hi @Lockless404, 

# APPROVED 🟢 

_Outstanding implementation of all the requirements of this milestone✔️_

You have done an amazing job. 🎉 

Kudos to you 💯 

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

## CONGRATULATIONS! 🎉 

# OPTIONAL SUGGESTIONS ♻️ 

Kindly check the comment below 👇 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Lockless404) [reply schema.sql:36]: Wow, I didn't think of that, an awesome approach. Thank you.

Comment (DammyShittu) [reply schema.sql:36]: You're welcome, @Lockless404 😀

**Pull request merged**